osu.Game.Rulesets.Mania/ManiaSettingsSubsection.cs

```diff
@@ -47,5 +46,10 @@ private void load()
                 }
             };
         }
+
+        private class ManiaScorllSlider : OsuSliderBar<double>
```

Comment:
Typo `ManiaScorllSlider` -> `ManiaScrollSlider`

Reply:
ouch
I'll fix it right away